tests/PageApprovalsIntegrationTest.php

```diff
@@ -4,16 +4,17 @@
 
 namespace ProfessionalWiki\PageApprovals\Tests;
 
-use MediaWiki\MediaWikiServices;
 use Title;
 use WikiPage;
 
 class PageApprovalsIntegrationTest extends \MediaWikiIntegrationTestCase {
 
-	protected function getIdOfExistingPage( string $titleText, string $text = 'Whatever wikitext' ): int {
-		$title = Title::newFromText( $titleText );
-		$this->editPage( $title, $text );
-		return MediaWikiServices::getInstance()->getWikiPageFactory()->newFromTitle( $title )->getId();
+	protected function createPageWithText( string $text = 'Whatever wikitext' ): WikiPage {
```

Comment:
Not sure yet if we need to merge this with `createPageWithCategories()`. So far we haven't had the need to set both text and categories, but it should be trivial to just add $text as an optional parameter.